lib/excoveralls/poster.ex

```diff
@@ -51,6 +51,9 @@ defmodule ExCoveralls.Poster do
            body
          })."}
 
+      {:error, :timeout} ->
+        {:ok, "Unable to upload the report to '#{endpoint}' due to a timeout. Not failing the build."}
```

Comment:
A bit unsure of the wording of this error message, would appreciate some feedback/suggestions.